internal/host/validator.go

```diff
@@ -957,6 +957,33 @@ func comparisonBuilder(value float64) string {
 	return equals
 }
 
+func (v *validator) generatePacketLossAdvisoryMessageForHost(c *validationContext, hostPacketLoss [] string) string {
+	message := "Actions:\n"
+	message += "1: Check if there are multiple devices on the same L2 network, if so then create a bond or disable all but one of the interfaces.\n"
```

Comment:
As a customer unfamiliar with our product, they might simply reach out to log in to the host and set up all this networking manually on the live environment - so perhaps we should tell them how they can go about setting up bonds or disabling interfaces etc (point them at the relevant nmstate networking config feature of AI)

Reply:
The exact phrasing of these directions should take into account both UI users and API users, as both of them might see this message

Reply:
IMO the wording here serves the purpose of this PR.
There's only so much we can cram in one error, and different users want to act upon this case of two interfaces on the same L2 network (which could be validation on its own) differently, as you mentioned.

Say we do point users to some URL (example: https://nmstate.io/examples.html#interfaces-bond), they might end up copying the configs as-is, and it would not work for them.

@paul-maidment, is this case failing the validation or just producing a warning?

Reply:
It is important to note that we have disabled the packet loss validation in prod (/cc @romfreiman @gamli75). The reasons for doing so are probably not valid any more, but we should be aware that whatever the PR produces here, won't be immediately exposed to users.

In case we re-enable the validation, it should be a blocking one.

Reply:
When a user sees "create a bond or disable all but one of the interfaces" - the only way a new user might know how to perform the action mentioned in that quote is by logging into the host and actively disabling the interface. This will be fruitless as it will only solve the issue during discovery and the issue will return once the host reboots.

We need to tell them they have to perform network configuration using our nmstate API / UI, regenerate the ISO, etc. All of that stuff is not at all obvious. I don't think we need to cram an entire tutorial inside the error message, just point the user at the relevant documentation for our networking config feature

Reply:
I couldn't agree more with Omer about the above. Also, is it worth to redact it all inside this error message given that we are going to spend the whole next week in rewriting our docs? Sounds like potentially a duplicated work

Reply:
So to clarify, is the suggestion that we include a link to the markdown files here?

https://github.com/openshift/assisted-service/blob/master/docs/user-guide/network-configuration/static-configuration.md

Reply:
In a few days/weeks we will have this markdown file rewritten and published directly in docs.openshift.com; my personal suggestion is to wait till it's done and then link to the official documentation

Reply:
@avishayt what do you think?

Reply:
I agree that some will configure it manually via SSH and that more information would be helpful.
However, linking to a doc isn't a great solution IMO because the URL can break and because we would need to point to different docs depending on SaaS, ACM, MCE, agent-based, etc.
How about something more generic like "..., if so then use the built-in nmstate-based advanced networking configuration to create a bond or disable all but one of the interfaces"

Reply:
i like that idea and agree with the point about the links

Reply:
I am for the links too